force-app/main/default/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -278,6 +290,35 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
 
     }
 
+    /**
+     * @description Test conditions and enqueue queueable job to clear Rollup totals.
+     */
+    @TestVisible
+    private void clearUnhandledRollupTotals() {
+        if (handledParentRecordIds.size() != recordIds.size()) {
+            if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
```

Comment:
If we're over the limit for whatever reason, does this functionality just not run?

Reply:
> If we're over the limit for whatever reason, does this functionality just not run?

Yes, that's correct.  I don't believe it is safe to do anything else.  That said, we should never be over the limit unless someone adds a queueable invocation in the future.